sdk-actors/src/main/java/io/dapr/actors/ActorUtils.java

```diff
@@ -13,8 +13,14 @@
 
 package io.dapr.actors;
 
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
 public final class ActorUtils {
 
+  private static String sdkVersion = null;

```

Comment:
Actor sdk import sdk module. 
So can't we just use Version.getSdkVersion() command from there?